mkdocs.yml

```diff
@@ -32,4 +32,5 @@ nav:
         - 'SQL Dictionary': 'wiki/99 - Developers/SQL Dictionary.md'
         - 'Automated Testing': 'wiki/99 - Developers/Automated Testing.md'
         - 'Style Guide (for help text)': 'HelpStyleGuide.md'
+        - 'Enabling reCAPTCHA': 'wiki/99 - Developers/reCAPTCHA.md'
```

Comment:
Why is this in the developers section? Shouldn't it be in the installation and configuration section? It's more for prod than devs..

Reply:
Yeah good point